db/seeds.rb

```diff
@@ -1,166 +1,23 @@
-    code: "WELDU",
-    name: "Welsh LDU",
-    email_address: "WalesNPS@example.com"
-)
-
-ldu2 = LocalDivisionalUnit.find_or_create_by!(
-    code: "ENLDU",
-    name: "English LDU",
-    email_address: "EnglishNPS@example.com"
-)
-
-ldu3 = LocalDivisionalUnit.find_or_create_by!(
-    code: "OTHERLDU",
-    name: "English LDU 2",
-    email_address: nil
-)
-
-team1 = Team.find_or_create_by!(
-    code: "WELSH1",
-    name: "NPS - Wales",
-    shadow_code: "W01",
-    local_divisional_unit: ldu1
-)
-
-team2 = Team.find_or_create_by!(
-    code: "ENG1",
-    name: "NPS - England",
-    shadow_code: "E01",
-    local_divisional_unit: ldu2
-)
-
-team3 = Team.find_or_create_by!(
-    code: "ENG2",
-    name: "NPS - England 2",
-    shadow_code: "E02",
-    local_divisional_unit: ldu3
-)
-
-# TODO: Change these CaseInformation records to use 'new' LDUs once the 'old' are removed
-# The offenders below are those with release dates a few years in the future and can therefore use the
-# responsibility override workflow
-
-CaseInformation.find_or_create_by!(nomis_offender_id: 'G7658UL') do |info|
-  info.tier = 'A'
-  info.case_allocation = 'NPS'
-  info.manual_entry =  true
-  info.team_id = team1.id
-  info.probation_service = "Wales"
-end
-
-CaseInformation.find_or_create_by!(nomis_offender_id: 'G7517GF') do |info|
-  info.tier = 'B'
-  info.case_allocation = 'NPS'
-  info.manual_entry = true
-  info.team_id = team2.id
-  info.probation_service = "Wales"
-end
-
-# 3 Test offenders which have handovers in Dec 2020
-['G1176UT', 'G0228VG', 'G1289UN'].each do |offender_no|
-  CaseInformation.find_or_create_by!(nomis_offender_id: offender_no) do |info|
-    info.assign_attributes(tier: 'B',
-                           case_allocation:'CRC',
-                           probation_service: 'Wales',
-                           manual_entry: true,
-                           team: team2)
+# This is live seed data so it is to be kept up to date if new prisons are added or modified
+
+def prison_type(prison)
+  if prison.gender == :male
+    if PrisonService::OPEN_PRISON_CODES.include?(prison.code)
+      'mens_open'
+    else
+      'mens_closed'
+    end
+  else
+    'womens'
   end
 end
 
-# Test offenders which have handovers in Feb 2021
-['G2407UH', 'G5884GU'].each do |offender_no|
-  CaseInformation.find_or_create_by!(nomis_offender_id: offender_no) do |info|
-    info.assign_attributes(tier: 'B',
-                           case_allocation:'NPS',
-                           probation_service: 'Wales',
-                           manual_entry: true,
-                           team: team2)
+PrisonService::PRISONS.values.each do |prison|
+  # If prison data is already present it runs update instead to prevent a crash
+  if Prison.where(code: prison.code).exists?
+    Prison.update(code: prison.code, name: prison.name, prison_type: prison_type(prison))
+  else
+    Prison.create!(code: prison.code, name: prison.name, prison_type: prison_type(prison))
```

Comment:
Is the intention that this runs on every deployment in production environments?

This used to only be run manually by devs setting up their local installs – so I'm not sure that it's automatically running on production deploys.